scrapers_next/ar/people.py

```diff
@@ -131,3 +132,137 @@ def process_item(self, item):
+
+        
+        image_container = CSS(".col-md-4").match(self.root)[0]
+        image = XPath("//img[contains(@alt, '" + name + "')]").match(image_container)[0].get('src')
+        image = image.split("?")[0]
+
+        biocontainer = CSS(".panel-body").match(self.root)[0]
+        bio = XPath("//a/@href").match(biocontainer)[0]
+        committees = XPath("//div[contains(@class,base-text)]/ul/li").match(self.root)
+        
+        
+
+        table = {
+            "Party": "",
+            "Phone": "",
+            "Email": "",
+            "District": "",
+            "Seniority": "",
+            "Occupation": "",
+            "Party": "",
+            "District Address": "",
+            "Church Affiliation:": "",
+            "Veteran": "",
+            "Public Service": "",
+            "Legislative Service": "",
+            "Biography": "",
+        }
+
+        for row in rows:
+            try: 
+                text = row.text_content()
+                text = text.split(": ")
+                if (len(text) < 2):
+                    continue
+                name = text[0]
+                value = text[1]
+                table[name] = value
+                if name == "Legislative Service":
+                    if (value[0] != 'H'):
+                      value.replace("House", "; House")
+                    table[name] = value
+                    break
+
+            except SelectorError:
+                pass
+
+        table["Biography"] = bio
+
+        p = ScrapePerson(
+            name=name,
+            state="ar",
+            chamber=self.input.chamber,
+            party=table["Party"],
+            image=image,
+            district=table["District"],
+            email=table["Email"],
+        )
+
+        committees_string = ""
+        first_committee = True;
+        for c in committees:
+            found = False
+            for k in table.keys():
+                if c.text_content().find(k) >= 0:
+                   found = True
+            if not found:
+                if not first_committee:
+                    committees_string += ", "
+                committees_string += c.text_content().lower()
+                first_committee = False;
+        p.extras["Committees"] = committees_string
+
+        if table["Phone"] != "":
+            p.district_office.voice = table["Phone"]
+        if table["Biography"] != "":
+            p.add_link(table["Biography"], "Biography")
+
+        for key in table:
+            if (
+                key == "Phone"
+                or key == "Email"
+                or key == "District"
+                or key == "Biography"
+            ):
+                continue
+            elif table[key] != "":
+                # remove the colon at the end
+                p.extras[key.lower()] = table[key]
+        try:
+            address = CSS(".col-md-12 p b").match_one(self.root).text_content()
```

Comment:
Wouldn't you be able to use `address = table["District Address"]` here instead of getting it off the page again?